src/test/java/bio/terra/service/dataset/DatasetDaoTest.java

```diff
@@ -904,6 +904,19 @@ public void createDatasetWithProperties() throws Exception {
     datasetDao.delete(datasetId, TEST_USER);
   }
 
+  @Test
+  public void updatePredictableFileIdsFlag() throws Exception {
+    UUID datasetId = createDataset("dataset-minimal.json");
+    assertFalse(
+        "predictable file ids flag is false",
+        datasetDao.retrieve(datasetId).hasPredictableFileIds());
+    datasetDao.setPredictableFileId(datasetId, true);
+    assertTrue(
+        "predictable file ids flag is false",
```

Comment:
I think this should be "predictable file ids flag is true" :)

Reply:
sigh, you are right...will update